paddle/fluid/operators/conv_op.cc

```diff
@@ -209,6 +209,12 @@ void Conv2DOpMaker::Make() {
       .SetDefault(false);
   AddAttr<bool>("fuse_relu", "(bool, default false) Only used in mkldnn kernel")
       .SetDefault(false);
+  AddAttr<bool>("fuse_brelu",
+                "(bool, default false) Only used in mkldnn kernel")
+      .SetDefault(false);
+  AddAttr<float>("fuse_brelu_threshold",
```

Comment:
Why need `fuse_brelu_threshold`?

Reply:
> Why need `fuse_brelu_threshold`?

It's a parameter of bounded ReLU. 6 is the typical value but it may vary.

Reply:
@luotao1 , `relu6` is a case of `bounded_relu` op in which 6 is the default value of the threshold parameter.

Reply:
Hi, when will this PR be merged ? :)

Reply:
@qingqing01 How do you see this PR add two attributes `fuse_brelu` and `fuse_brelu_threshold`?